packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -633,12 +634,20 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
 			}
 		});
 
-		await super.initialize(connectMessage, timeout).finally(() => {
+		try {
+			await super.initialize(connectMessage, timeout);
+			if (this._details) {
+				if (!this.details.supportedFeatures) {
+					this._details.supportedFeatures = {};
+				}
+				this._details.supportedFeatures![enable_single_commit_summary] = true;
+			}
+		} finally {
 			this.logger.sendTelemetryEvent({
```

Comment:
Are there known cases where `initialize` throws? Curious why you're adding the try-finally - seems like a good addition though.